.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
```

Comment:
This currently just carries over the wording from the existing issue template. I can see that it might come across as condescending, but it does have the advantage of being forceful and concise -- which were the main motivations for switching to the existing wording in #107150.

> what if they need help with a bug?

Then they should _probably_ try to get that help on Discourse :)

I'll try to make it sound less condescending, but I think concision is really important here -- my impression is that people just weren't reading the issue template prior to #107150 (and they probably still aren't reading it now tbh).

Reply:
Make sense, and the current wording is much better IMHO.

> Then they should probably try to get that help on Discourse :)

This actually sounds like what I just suggested in the previous message: they should probably ask on Discourse/python-list even if they *think* they found a bug, not just if they are new to Python and need help/advice (the wording currently doesn't seem to include them).